File: python/8kyu/triple_trouble.py
def triple_trouble(one, two, three):
    x = []
    y = []
    z = []
    lis = []
    
    [x.append(i) for i in one]
    [y.append(i) for i in two]
    [z.append(i) for i in three]

    while x or y or z:
      lis.append(x.pop(0))
      lis.append(y.pop(0))
      lis.append(z.pop(0))
    
    print("".join(lis))
    


one = "LLh"
two = "euo"
three = "xtr"
triple_trouble(one, two, three)
    

File: javascript/8_kyu/triple_trouble.js
// Triple Trouble
// https://www.codewars.com/kata/5704aea738428f4d30000914/train/javascript

function tripleTrouble(one, two, three){
  var ans = [];
  for (var i = 0; i < one.length; i++) {
    ans.push(one[i]);
    ans.push(two[i]);
    ans.push(three[i]);
  }
  
  return ans.join('')
 }